.travis.yml

```diff
@@ -5,7 +5,7 @@ jdk:
 
 script:
   # tests are not done with mvn test - dont know why
-  - mvn verify cobertura:cobertura
+  - ./scripts/build_mvn.sh
```

Comment:
How about moving it to Makefile like frontend did?

Reply:
I don't like Makefile's